process.md

```diff
@@ -27,34 +20,47 @@ We want to avoid deincentivising authors who opt into the reproducibility challe
+
+## Requirements for reviewers
+We require a minimum level of practical experience in climate data science to participate as a reviewer. This ensures that the review process can focus on technical detail rather than computational literacy. Reviewers will be required to evidence their experience through their own software publications. Full reviewer requirements will be published ==XXX==
 
 ## Expected workload
 
-- 16 submissions, 2 reviewers per AE submission, 2 submissions per reviewer (so AEC of up to 16 reviewers)
+- Each reviewer will be assigned 2 submissions
 - 2-hour on-boarding session to get everyone up to speed
+- Each review can be expected to take ==XXX== hours to complete, including direct communications with the submitting author via our review portal
+- Final reviews should be submitted by ==XXX==
+
+## Benefits to reviewers
+Reviewers will benefit from hands-on training in high fidelity computational reproducibility, which we anticipate will support their own development of reproducible research artefacts. Reviewers will be able to reference their contribution as evidence of leadership culture change towards reproducibility, and invited to co-author a [retrospective report](#retrospective-report) to be published in [Environmental Data Science](https://www.cambridge.org/core/journals/environmental-data-science) after the review process is complete. They will be [supported](#support) by the CI Reproducibility team and AEC throughout, thereby strengthening their connections with this highly skilled team.
+
+##  Evaluation process
+
+Reviewers will asses the artefacts against a checklists provided for each ["badge" level to be awarded](https://github.com/alan-turing-institute/climate-informatics-2024-ae/blob/main/badges.md).
 
-# Submitter guidelines
+For each point, reviewers will be required to briefly note the evidence for this point, anything they have done to validate that point (e.g., what did you need to reproduce a claim), as well as the outcome (negative, neutral, or positive) and a brief reason for their judgment.
+
+## Feedback to the authors
+
+This process has been designed to be collaborative and iterative. During the first 3 weeks, reviewers will be able to feed-back to authors.
+Authors will be encouraged to push improvements to the artefact whenever possible. For example, if essential documentation is missing, then rather than submitting a negative evaluation or providing only the reviewers with the required information, the authors should provide that information via a versioned updated to the artefact.
```

Comment:
```suggestion
Authors will be encouraged to push improvements to the artefact whenever possible. For example, if essential documentation is missing, then rather than submitting a negative evaluation or providing only the reviewers with the required information, the authors should provide that information via a versioned/updated artefact.
```